src/components/AuthenticatedRoute/index.jsx

```diff
@@ -31,4 +31,4 @@ const mapStateToProps = (state) => ({
     currentUser: state.user.currentUser,
 });
 
-export default connect(mapStateToProps, { FetchUserMe })(AuthenticatedRoute);
+export default connect(mapStateToProps, { FetchUserMe: fetchUserMe })(AuthenticatedRoute);
```

Comment:
Actually it was everywhere but as I said, camelCase, so I refactored that as well. Missed this file. Also I assumed this wouldnt solve that prop warning as because of this the component will still get FetchUserMe in prop.